src/components/forms/CharacterCount/CharacterCount.stories.tsx

```diff
@@ -5,14 +5,14 @@ import { FormGroup } from '../FormGroup/FormGroup'
 import { Label } from '../Label/Label'
 
 export default {
-  title: 'Components/Form controls/CharacterCount',
+  title: 'Components/CharacterCount',
```

Comment:
@suzubara I know you made an effort to align the organization of our storybook with USWDS. They've since reverted to un-nesting the form controls. Are you also good with us matching their organization again?

Reply:
Yes that's fine! let's match uswds